h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -116,6 +116,44 @@ describe('CreateGroupForm', () => {
     });
   });
 
+  describe('name field', () => {
+    it("doesn't go into an error state when too few characters are only input but not committed", () => {
+      const { wrapper, elements } = createWrapper();
+      const fieldEl = elements.name.fieldEl;
+
+      fieldEl.getDOMNode().value = 'aa';
+      fieldEl.simulate('input');
+
+      const { counterEl, fieldComponent } = getElements(wrapper).name;
+      assert.isNotOk(counterIsInErrorState(counterEl));
+      assert.isNotOk(componentIsInErrorState(fieldComponent));
+    });
+
+    it('goes into an error state when too few characters are committed', () => {
+      const { wrapper, elements } = createWrapper();
+      const fieldEl = elements.name.fieldEl;
+
+      // Too few characters are entered into the field and the field is then
+      // "committed" (e.g. the focus leaves the field, or the form is
```

Comment:
One detail I noticed when testing is that a `change` event is not emitted when focus leaves the field if the current value is the same as the previously committed value. This doesn't have an impact here, but I think is useful to be aware of.